.github/workflows/rust.yml

```
@@ -0,0 +1,44 @@
+        working-directory: ${{ matrix.app-dir }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          components: clippy, rustfmt
+
+      - name: Security audit
+        uses: actions-rs/audit-check@v1
```

actionlint

the runner of "actions-rs/audit-check@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)